<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingWdg</class>
 <widget class="QWidget" name="ProcessingWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Spatial filter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="drawPolygonBtn">
        <property name="text">
         <string>Draw</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clearPolygonBtn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="moduleCombo">
     <item>
      <property name="text">
       <string>Decluster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Completeness</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Recurrence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maximum magnitude</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="algorithmStacked">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="DeclusterWdg" name="declusterWdg"/>
     <widget class="CompletenessWdg" name="completenessWdg"/>
     <widget class="RecurrenceWdg" name="recurrenceWdg"/>
     <widget class="MaximumMagnitudeWdg" name="maxMagnitudeWdg"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeclusterWdg</class>
   <extends>QWidget</extends>
   <header>..decluster_wdg</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompletenessWdg</class>
   <extends>QWidget</extends>
   <header>..completeness_wdg</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaximumMagnitudeWdg</class>
   <extends>QWidget</extends>
   <header>..maximum_magnitude_wdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecurrenceWdg</class>
   <extends>QWidget</extends>
   <header>..recurrence_wdg</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>moduleCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>algorithmStacked</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
